src/core/server/bootstrap.ts

```diff
@@ -11,32 +11,20 @@ import { CliArgs, Env, RawConfigService } from './config';
 import { Root } from './root';
 import { CriticalError } from './errors';
 
-interface KibanaFeatures {
-  // Indicates whether we can run Kibana in dev mode in which Kibana is run as
-  // a child process together with optimizer "worker" processes that are
-  // orchestrated by a parent process (dev mode only feature).
-  isCliDevModeSupported: boolean;
-}
```

Comment:
I assumed we were not planning to use this `features` param anytime soon now that we got rid of the `isCliDevModeSupported` feature/flag, so I deleted it completely.